### ISSUE TITLE
ingress creation: tls "true" is used as secret name

### ISSUE THREAD

**Issue by shaunc** [devspace-cloud/devspace#912]
**What happened?**  

The documentation says that `tls: true` will use an auto-generated secret name. However, it actually generates an ingress with secret named "true". For example:
```
version: v1beta6
deployments:
- name: foo-ingress
  helm:
    componentChart: true
    values:
      ingress:
        rules:
        - host: a.b.c
          path: '/foo/(.*)'
          serviceName: foo
          servicePort: 443
          tls: true
        annotations:
          kubernetes.io/ingress.class: "nginx"
          cert-manager.io/cluster-issuer: "letsencrypt-dev"
          nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
          nginx.ingress.kubernetes.io/rewrite-target: "/$1"
```
generates an ingress:
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt-dev
    helm.sh/chart: component-chart-v0.0.8
    kubernetes.io/ingress.class: nginx
    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
    nginx.ingress.kubernetes.io/rewrite-target: /$1
  creationTimestamp: "2020-02-09T23:16:00Z"
  generation: 1
  labels:
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: foo-ingress
  name: foo-ingress
  namespace: crane-web
  resourceVersion: "1788051"
  selfLink: /apis/extensions/v1beta1/namespaces/crane-web/ingresses/foo-ingress
  uid: 22bae142-4b92-11ea-83ad-0e52da6d9e55
spec:
  rules:
  - host: a.b.c
    http:
      paths:
      - backend:
          serviceName: foo
          servicePort: 443
        path: /foo/(.*)
  tls:
  - hosts:
    - a.b.c
    secretName: "true"
status:
  loadBalancer: {}
```

**What did you expect to happen instead?**  

The "secretName" field should reference a more-appropriately-named autogenerated secret.

**How can we reproduce the bug?** (as minimally and precisely as possible)  

Should be enough above.

**Local Environment:**  
- Operating System: mac
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: aws
- Kubernetes Version:

Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T23:41:55Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.9-eks-c0eccc", GitCommit:"c0eccca51d7500bb03b2f163dd8d534ffeb2f7a2", GitTreeState:"clean", BuildDate:"2019-12-22T23:14:11Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}

**Anything else we need to know?**  

/kind bug

**Comment by LukasGentele** [comment]
Thanks for reporting this @shaunc. Should be fixed with v0.1.1: https://github.com/devspace-cloud/devspace/issues/900#issuecomment-583561471

Will be fixed for componentChart:true with the next DevSpace release as well.

**Comment by shaunc** [comment]
Excellent! :)
